regolith/helpers/a_proposalhelper.py

```diff
@@ -28,70 +23,57 @@ def subparser(subpi):
     subpi.add_argument("name", help="A short but unique name for the proposal",
                        default=None
                        )
-    subpi.add_argument("-amt", "--amount", help="value of award",
+    subpi.add_argument("amount", help="value of award",
                        default=None
                        )
-    subpi.add_argument("-a", "--authors",
-                       help="Other investigator names"
+    subpi.add_argument("authors", help="Other investigator names"
```

Comment:
this will need an ``nargs='+'`` so a list can be input

Reply:
done